main.cpp

```diff
@@ -386,6 +437,19 @@ struct StudioLighting
     {
         std::cout << "Studio Lighting Destructor" << std::endl;
     }
+
+    JUCE_LEAK_DETECTOR(StudioLighting)
+};
+
+struct StudioLightingWrapper
+{
+    StudioLightingWrapper(StudioLighting* StudioLightingPointer) : studioLightingptr(StudioLightingPointer){}
+    ~StudioLightingWrapper()
+    {
+        delete studioLightingptr;
+    }
+    
+    StudioLighting* studioLightingptr = nullptr;
```

Comment:
`camelCase` your member variable name.  
Type names begin with capital letters.  variables do not.
- [ ] revise